llvm/lib/Target/AArch64/AArch64SchedPredNeoverse.td

```diff
@@ -80,5 +80,22 @@ def NeoverseZeroMove : MCSchedPredicate<
                            // MOVI Dd, #0
                            // MOVI Vd.2D, #0
                            CheckAll<[CheckOpcode<[MOVID, MOVIv2d_ns]>,
-                                     CheckImmOperand<1, 0>]>
+                                     CheckImmOperand<1, 0>]>,
+                           // MOV Zd, Zn
+                           CheckAll<[CheckOpcode<[ORR_ZZZ]>,
+                                     CheckSameRegOperand<1, 2>]>,
+                           // MOV Vd, Vn
+                           CheckAll<[CheckOpcode<[ORRv16i8, ORRv8i8]>,
+                                     CheckSameRegOperand<1, 2>]>,
+                         ]>>;
+
+def NeoverseAllElement : MCSchedPredicate<
```

Comment:
Any objections to naming this `NeoverseAllActivePredicate`? We have similarly named methods in ISel and TTI to check for all-active predicates.